staging/vhost-device-gpu/rutabaga_gfx/src/rutabaga_core.rs

```diff
@@ -50,6 +50,12 @@ pub struct RutabagaResource {
     pub size: u64,
     pub mapping: Option<MemoryMapping>,
 }
+// SAFETY: Safe as the structure can be sent to another thread.
+unsafe impl Send for RutabagaResource {}
+
+// SAFETY: Safe as the structure can be shared with another thread as the state
+// is protected with a lock.
+unsafe impl Sync for RutabagaResource {}
```

Comment:
How sure are you about the `// SAFETY` comments here? It seem a bit scary, considering this comment for example:
https://github.com/dorindabassey/vhost-device/blob/f194b995b26b2115ab101318a088f4423505ba59/staging/vhost-device-gpu/rutabaga_gfx/src/rutabaga_core.rs#L35-L36

Reply:
one way or another, we have to find a way to adapt the rutabaga crate to our vhost-device backend structure, the `VhostUserBackendMut: Send + Sync` bound needs it to be send and sync. eventually we would have our own tweaked version of rutabaga.